ansible/roles/test/files/ptftests/arista.py

```diff
@@ -520,3 +526,21 @@ def check_change_time(self, output, entity, what):
         # Note: the first item is a placeholder
         return 0, change_count
 
+    def get_bp_ip(self, v4=True):
+        if v4:
+           cmd = 'show ip route | grep \'S\' | grep \'Ethernet5\' | head -n 1'
```

Comment:
>Ethernet5 [](start = 54, length = 9)

Why `Ethernet5` ? What if DUT has no `Ethernet5` ?

Reply:
I will get rid of this function and get the nexthop ips from the topo file

Reply:
I will get rid of this function and get the nexthop ips from the topo file

Reply:
done